src/OpenTK/Math/Vector2d.cs

```diff
@@ -627,6 +631,118 @@ public static void Max(ref Vector2d a, ref Vector2d b, out Vector2d result)
 
         #endregion
 
+        #region ComponentMin
+
+        /// <summary>
+        /// Calculate the component-wise minimum of two vectors
+        /// </summary>
+        /// <param name="a">First operand</param>
+        /// <param name="b">Second operand</param>
+        /// <returns>The component-wise minimum</returns>
+        public static Vector2d ComponentMin(Vector2d a, Vector2d b)
```

Comment:
We might want to define exactly what we mean by component-wise in a more explicit way in the documentation.
Based on the name, this could be interpreted to mean "Choose the vector with the absolute smallest of any components."
Something like "returns a vector given by the smallest of each component from vectors `a and b`"